src/main/docker/Dockerfile

```diff
@@ -15,8 +15,11 @@
 #
 
 ARG DOCKER_HUB_PUBLIC=docker.io
-FROM ${DOCKER_HUB_PUBLIC}/digitallyseamless/nodejs-bower-grunt:5
+FROM ${DOCKER_HUB_PUBLIC}/cafapi/opensuse-nodejs10:latest
 
+RUN zypper install -y tar && \
```

Comment:
If you're now installing `tar` using `zypper`, then do we still need to download and compile it just below?